sdk/storage/azure-storage-blob/src/main/java/com/azure/storage/blob/sas/BlobServiceSasSignatureValues.java

```diff
@@ -702,7 +701,7 @@ private String stringToSign(String canonicalName) {
             this.identifier == null ? "" : this.identifier,
             this.sasIpRange == null ? "" : this.sasIpRange.toString(),
             this.protocol == null ? "" : this.protocol.toString(),
-            version, /* Pin down to version so old string to sign works. This is reflected in the declaration of
```

Comment:
Should we hardcode this? This is all called from deprecated code and I feel like this will break as the string to sign changes

Reply:
That's a good idea. Currently this code doesn't work... that string to sign (or the other one below/above) hasn't been changed as service version evolved